### PR TITLE
[ADD] sale_order_product_template: Product template field in sale rep…

### PR DESCRIPTION
**Pull request by Berezi**
…ort pivot view.

### CODE REVIEW

**Comment by Berezi** [comment]
@anajuaristi @oihane @Daniel-CA @alfredoavanzosc review please.

File: sale_report_product_template/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2021 Berezi - AvanzOSC
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Sale Report Product Template",
+    'version': '12.0.1.0.0',
+    "author": "Avanzosc",
```
Comment (Daniel-CA) [inline sale_report_product_template/__manifest__.py]: AvanzOSC

Comment (Berezi) [reply sale_report_product_template/__manifest__.py]: Zuzenduta. Eskerrik asko.

**Review by Daniel-CA — APPROVED** [review]

**Review by alfredoavanzosc — APPROVED** [review]
:+1:

**Pull request merged**